frontend/src/App.vue

```diff
@@ -116,14 +154,21 @@
     <div v-if="showCookieMsg" id="cookies" class="has-background-grey">
       <div class="column has-text-centered p-1">
         <div class="has-text-white">
-          We use cookies to enhance the usability of our website.
-          By continuing you are agreeing to our
-          <router-link class="has-text-white has-text-weight-bold"
-                       :to="{ name: 'about', hash: '#privacy' }">
-            Privacy Notice and Terms of Use
-          </router-link>&emsp;
-          <p class="button is-small is-rounded has-background-danger has-text-white has-text-weight-bold"
-             @click="showCookieMsg=false; acceptCookiePolicy()">
+          We use cookies to enhance the usability of our website. By continuing you are agreeing to
+          our
+          <router-link
+            class="has-text-white has-text-weight-bold"
+            :to="{ name: 'about', hash: '#privacy' }"
+          >
+            Privacy Notice and Terms of Use </router-link
+          >&emsp;
```

Comment:
This looks weird, I was expecting:
```suggestion
            Privacy Notice and Terms of Use
          </router-link>
          &emsp;
```

Reply:
This does look weird and it happens at multiple places. 
I did a quick search with the command 
```$ grep -irn "[a-z].*</router-link" .``` in the folder `frontend/src` and found quite a few examples. 

```
./components/SearchTable.vue:53:              <span class="suggestions">{{ v }}</span> </router-link
./components/SearchTable.vue:187:                        >{{ sub.name }}</router-link
./components/SearchTable.vue:209:                          >{{ comp.name }}</router-link
./components/About.vue:44:            >Human-GEM</router-link
./components/About.vue:48:            >Yeast-GEM</router-link
./components/About.vue:82:          <router-link :to="{ name: 'gems' }">GEM Repository</router-link> for links to those.<br /><br />
./components/explorer/gemBrowser/Reaction.vue:41:                  {{ v.name }}</router-link
./components/explorer/gemBrowser/Reaction.vue:55:                      >{{ c.name }}</router-link
./components/explorer/gemBrowser/Reaction.vue:86:                {{ rr.id }} </router-link
./components/explorer/gemBrowser/Subsystem.vue:37:                    >{{ c.name }}</router-link
./components/explorer/gemBrowser/Metabolite.vue:43:                >{{ metabolite[el.name].id }}</router-link
./components/Repository.vue:67:          <router-link :to="{ name: 'documentation', hash: '#FTP-access' }">FTP server</router-link
./components/Repository.vue:179:                    >FTP server</router-link
./components/Documentation.vue:93:              >GEM Repository</router-link
./components/Documentation.vue:510:              >GEM Repository</router-link
./components/NotFound.vue:31:            <b>new version of Metabolic Atlas</b> </router-link
./components/NotFound.vue:34:          <router-link :to="{ name: 'gems' }"><b>GEM > Repository</b></router-link>
./App.vue:163:            Privacy Notice and Terms of Use </router-link
```

Reply:
Fixed here: https://github.com/MetabolicAtlas/MetabolicAtlas/pull/797/commits/a21f080b80e8b59f661b064710407d0b5b947a34#diff-347c85641aa3243ab418a0dfededf202763a1d5a28dc34eacb00b912b1a97fe4R5